### DISCUSSION TITLE
Remember accessory state after reset!

### DISCUSSION THREAD

**Discussion by rakeshnsw** [HomeSpan/HomeSpan#599] [Q&A]
I noticed that HomeSpan doesn't remember the last accessories state after ESP reset. Is there any configuration setting to achieve that?

**Comment by HomeSpan** [comment]
You can turn this on for any Characteristic by setting the second parameter of the constructor, *nvsStore* to **true** (the default is **false**).  See the [API](https://github.com/HomeSpan/HomeSpan/blob/master/docs/Reference.md#spancharacteristicvalue-boolean-nvsstore) for details.

**Comment by rakeshnsw** [comment]
Perfect 👍🏻 And once again, thanks for the quick response 🙏🏻 ☺️

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
